### PR TITLE
Update dlp-configure-endpoint-settings.md

### LINKED ISSUE #12018: Broken hyperlink
image

Broken link for requirement of **Network share coverage and exclusions (preview)**
**Microsoft Defender April-2023 (Platform: 4.18.2304.8 | Engine: 1.1.20300.3)**
---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: 34f25ec8-d5a8-4a9d-ca72-bd61eacd9dbc
* Version Independent ID: 34f25ec8-d5a8-4a9d-ca72-bd61eacd9dbc
* Content: [Configure endpoint DLP settings - Microsoft Purview (compliance)](https://learn.microsoft.com/en-us/microsoft-365/compliance/dlp-configure-endpoint-settings?view=o365-worldwide#network-share-coverage-and-exclusions-preview)
* Content Source: [microsoft-365/compliance/dlp-configure-endpoint-settings.md](https://github.com/MicrosoftDocs/microsoft-365-docs/blob/public/microsoft-365/compliance/dlp-configure-endpoint-settings.md)
* Service: **o365-seccomp**
* GitHub Login: @chrfox
* Microsoft Alias: **chrfox**

### PR DESCRIPTION
**Pull request by msbemba**
Fixed Broken link

Fixes https://github.com/MicrosoftDocs/microsoft-365-docs/issues/12018

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: microsoft-365/compliance/dlp-configure-endpoint-settings.md [hunk 1]
```diff
@@ -148,7 +148,7 @@ DLP can copy items that match policies on devices to an [Azure storage account](
 > - Windows 10 - [March 21, 2023—KB5023773 (OS Builds 19042.2788, 19044.2788, and 19045.2788) Preview](https://support.microsoft.com/en-us/topic/march-21-2023-kb5023773-os-builds-19042-2788-19044-2788-and-19045-2788-preview-5850ac11-dd43-4550-89ec-9e63353fef23), [March 28, 2023—KB5023774 (OS Build 22000.1761) Preview](https://support.microsoft.com/en-us/topic/march-28-2023-kb5023774-os-build-22000-1761-preview-67b4cfda-120a-422f-98c0-35124ddba839) 
 >- Windows 11 - [March 28, 2023—KB5023778 (OS Build 22621.1485) Preview](https://support.microsoft.com/en-us/topic/march-28-2023-kb5023778-os-build-22621-1485-preview-d490bb51-492e-410c-871f-50ad01b0f765)
 >
-> - Microsoft Defender [April-2023 (Platform: 4.18.2304.8 | Engine: 1.1.20300.3)](/microsoft-365/security/defender-endpoint/microsoft-defender-antivirus-updates.md#april-2023-platform-41823048--engine-11203003)
+> - Microsoft Defender [April-2023 (Platform: 4.18.2304.8 | Engine: 1.1.20300.3)](/microsoft-365/security/defender-endpoint/microsoft-defender-antivirus-updates?view=o365-worldwide#april-2023-platform-41823048--engine-11203003)
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/dlp-configure-endpoint-settings.md]:
```suggestion
> - Microsoft Defender [April-2023 (Platform: 4.18.2304.8 | Engine: 1.1.20300.3)](/microsoft-365/security/defender-endpoint/microsoft-defender-antivirus-updates#april-2023-platform-41823048--engine-11203003)
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@tiburd Please merge this PR. Thanks

**Pull request merged**